docs/contribute/deployment.md

```diff
@@ -30,68 +30,56 @@ page_nav:
-
-<h2 id="gitlab-pages">GitLab Pages</h2>
-<div class="callout callout--info">
-<p><strong>What are GitLab Pages?</strong> With <a href="https://about.gitlab.com/features/pages/" target="blank">GitLab Pages</a> you can create static websites for your GitLab projects, groups, or user accounts. Connect as many customs domains as you like and bring your own TLS certificate to secure them.</p>
-</div>
-<div class="callout callout--info">
-<p><strong>Full GitLab Pages deployment guide.</strong> We recommend you to read full GitLab Pages deployment guide <a href="https://docs.gitlab.com/ee/user/project/pages/" target="blank">here</a>.</p>
-</div>
-<div class="callout callout--warning">
-<p><strong>Set relative URLs properly!</strong> Please take a look at Relative URLs part of "Configuration" section before you deploy your site.</p>
-</div>
-<div class="callout callout--warning">
-<p><strong>Build site with environment variable!</strong> Please build your site with proper environment variable before you deploy. Your build command should look like this <code>JEKYLL_ENV=production jekyll build</code>.</p>
-</div>
-
-<h3 id="user-and-organisation-pages-1">User and Organisation Pages</h3>
-<p>User and organisation pages live in a special GitLab repository dedicated to only the GitLab Pages files. This repository must be named after the account name.</p>
-
-<p>For example, if you create a project called <code class="highlighter-rouge">john.gitlab.io</code> under your username, <code class="highlighter-rouge">john</code>, your project URL will be <code class="highlighter-rouge">https://gitlab.com/john/john.gitlab.io</code>. Once you enable GitLab Pages for your project, your website will be published under <code class="highlighter-rouge">https://john.gitlab.io</code>.</p>
-
-<p>Content from the <code class="highlighter-rouge">gl-pages</code> branch of your repository will be used to build and publish the GitLab Pages site, so make sure your <code class="highlighter-rouge">_site</code> directory content is stored there.</p>
-
-<h3 id="project-pages-1">Project Pages</h3>
-<p>Unlike user and organisation pages, project pages are kept in the same repository as the project they are for, except that the website content is stored in a specially named <code class="highlighter-rouge">gl-pages</code> branch.</p>
-
-<p>Content from the <code class="highlighter-rouge">gl-pages</code> branch of your repository will be used to build and publish the GitLab Pages site, so make sure your <code class="highlighter-rouge">_site</code> directory content is stored there.</p>
-
-<p>Output will become available under a subpath of your user pages subdomain, such as <code class="highlighter-rouge">username.gitlab.io/project</code> (unless a custom domain is specified).</p>
-
-                </div><!-- /.content -->
-            </div><!-- /.col -->
-        </div><!-- /.row -->
-</div><!-- /.container -->
+## Web hosting providers (FTP)
```

Comment:
I feel like this content needs some introduction. Like "SingularityNET hosts the public community dev portal using github pages, and github automatically builds and deploys the site from the master branch. If you want to deploy a local or private copy, there are several options available."

Reply:
Edited this a bit and added it to an "Introduction" section.